web_touchscreen/static/src/fields.esm.js

```diff
@@ -0,0 +1,27 @@
+import {Field} from "@web/views/fields/field";
+import {hasTouch} from "@web/core/browser/feature_detection";
+import {patch} from "@web/core/utils/patch";
+import {utils} from "@web/core/ui/ui_service";
+
+patch(Field, {
+    /**
+     * @override
+     * Whenever a x2many field loads a list we want to force the kanban view over the
+     * list one if both options are available and a touchscreen is being used.
+     */
+    parseFieldNode(node) {
+        const fieldInfo = super.parseFieldNode(...arguments);
+        // Already handled in super
+        if (utils.isSmall()) {
+            return fieldInfo;
+        }
+        const viewMode = node.getAttribute("mode");
+        const kanban_is_optional =
+            (viewMode && viewMode.split(",").length !== 1) ||
+            (fieldInfo.views?.list && fieldInfo.views?.kanban);
+        if (fieldInfo?.viewMode === "list" && kanban_is_optional && hasTouch()) {
```

Comment:
issue: this will be redundant if you apply my other suggestion

```suggestion
        if (fieldInfo?.viewMode === "list" && kanban_is_optional) {
```